### PR TITLE
feat: experimental package + json query engine

### PR DESCRIPTION
**Pull request by EmanuelCampos**
fixing types not being generated

### CODE REVIEW

**Review by marcusschiesser — COMMENTED** [review]
Great job - looks good to me!

File: package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "release": "pnpm run build:release && changeset publish",
     "new-llamaindex": "pnpm run build:release && changeset version --ignore create-llama",
     "new-create-llama": "pnpm run build:release && changeset version --ignore llamaindex --ignore @llamaindex/core-test",
+    "new-experimental": "pnpm run build:release && changeset version --ignore create-llama",
```
Comment (marcusschiesser) [inline package.json:17]: @EmanuelCampos this operation is then the same as `new-llamaindex`, you probably want to ignore more or leave it out (which would mean experimental versioning would depend on core)

File: packages/experimental/package.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/run-llama/LlamaIndexTS.git",
+    "directory": "packages/experimental"
+  },
+  "scripts": {
+    "build": "rm -rf ./dist && pnpm run build:esm && pnpm run build:cjs && pnpm run build:type",
+    "build:esm": "swc src -d dist --strip-leading-paths --config-file .swcrc",
+    "build:cjs": "swc src -d dist/cjs --strip-leading-paths --config-file .cjs.swcrc",
```
Comment (himself65) [inline packages/experimental/package.json:50]: https://github.com/run-llama/LlamaIndexTS/pull/620

File: package.json (hunk 1 above)
Comment (EmanuelCampos) [reply package.json:17]: oh true hehe, mb

**Pull request merged**